### PR TITLE
Test files

### PR DESCRIPTION
**Pull request by sushmithaaredhatdev**
Fixes :#https://github.com/thoth-station/storages/issues/5

### CODE REVIEW

File: images/janusgraph/hack/files/indexes/byDependsOn.groovy [hunk 1]
```diff
@@ -7,26 +7,31 @@ graph.tx().rollback() //Never create new indexes while a transaction is active
 
 mgmt = graph.openManagement()
 
+if (mgmt.getGraphIndex("byDependsOn") == null) {
 
-/*
```
Comment (fridex) [inline images/janusgraph/hack/files/indexes/byDependsOn.groovy]: IMHO two new lines are not needed.

File: images/janusgraph/hack/files/indexes/byDependsOn.groovy [hunk 2]
```diff
@@ -7,26 +7,31 @@ graph.tx().rollback() //Never create new indexes while a transaction is active
...
 
-// No property edges match.
-mgmt.buildIndex('byDependsOn', Edge.class) \
-  .addKey(lbl) \
-  .addKey(type) \
-  .addKey(version_range) \
-  .addKey(package_name) \
-  .addKey(extras) \
-  .buildCompositeIndex()
+  // No property edges match.
```
Comment (fridex) [inline images/janusgraph/hack/files/indexes/byDependsOn.groovy]: Please fix indentation so the code is properly indented.

**Comment by fridex** [comment]
There will be a need to adjust code according to review comments. Please go through comments in #14, #15 and #16. They all have duplicate entries that confused me first.

**Comment by sushmithaaredhatdev** [comment]
Completed the minor above mentioned changes , please verify and let me know @fridex

**Pull request merged**